src/compiler/parser.ts

```diff
@@ -1310,7 +1310,11 @@ namespace ts {
             nextToken();

```

Comment:
nextTokenCanFollowDefaultKeyword doesn't really seem like an accurate name considering it's looking at multiple? Same with nextTokenCanFollowModifier. Would something like `declarationCanFollowExportDefault` be more appropriate?

Reply:
I think the name is fine since it doesn't take a `declaration` as input, it just goes to the next token and looks at its kind and *maybe* does some minimal lookahead.